src/openrct2/peep/Peep.cpp

```diff
@@ -3329,3 +3328,26 @@ void Peep::RemoveFromRide()
     }
     StateReset();
 }
+
+uint32_t Peep::GetItemFlags(bool ExtraItem) const
```

Comment:
I would make this function like the following:

For cases where you are checking an individual flag:
i.e. to replace `peep->ItemStandardFlags & PEEP_ITEM_BALLOON`
```
bool Peep::HasItem(ShopItem item) const 
{
    return GetItemFlags() & EnumToFlag(item);
}
```

For cases where you need to check a few flags at the same time (unsure if there are any cases of this but need for the above either way):
i.e. `peep->ItemStandardFlags & (PEEP_ITEM_BALLOON | PEEP_ITEM_VOUCHER)`
```
uint64_t Peep::GetItemFlags() const
{
    return ItemStandardFlags | (ItemExtraFlags << 32);
}
```

For instances where you need to clear a flag:
i.e. `peep->ItemStandardFlags &= ~PEEP_ITEM_BALLOON;`
```
void Peep::RemoveItem(ShopItem item)
{
    // TODO: Join up standard and extra flags into a `uint64_t` to remove this if for the extra flags
    if (EnumValue(item) > 31)
    {
        ItemExtraFlags &= ~(1 << (EnumValue(item) - 31));
    } 
    else
    {
        ItemStandardFlags &= ~EnumToFlag(item);
    }
}
```

Then I'd also rename your `ResetItemFlags` to `RemoveAllItems`